addons/im_livechat/models/im_livechat_channel.py

```diff
@@ -67,7 +67,7 @@ def _are_you_inside(self):
     @api.depends('user_ids.im_status')
     def _compute_available_operator_ids(self):
         for record in self:
-            record.available_operator_ids = record.user_ids.filtered(lambda user: user.im_status == 'online')
+            record.available_operator_ids = record.user_ids.filtered(lambda user: 'online' in user.im_status)
```

Comment:
this could be slightly less hacky since all online seems to be at the end

```suggestion
            record.available_operator_ids = record.user_ids.filtered(lambda user: user.im_status.rpartition('_')[2] == 'online')
```

Reply:
Wouldn't that break with stuff like 'leave_online' though ?